components/light/rgb.rst

```diff
@@ -60,6 +60,7 @@ perceived intensity of different colors will generally vary. This can be done by
         max_power: 80%
       # Repeat for green and blue output
 
+Remember that ``gamma_correct`` is enabled by default, and you may want take it into account for the calibration. For instance if you command a light to *50%* brightness and want it to be the new maximum: ``max_PWM_power = max_light_power^2.8 = 0.5^2.8 = 0.144``, then you would set ``max_power`` to *14.4%*.
```

Comment:
Please put this in a `note` box (probably better below the config variables too, since that's the usual place).

```rst
.. note::

    Remember that ...
```